### PR TITLE
Calculator design

### PR DESCRIPTION
**Pull request by BarryLev**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

The feature should be created for enabling old/new calculator design(old - from master, new - from develop)

## Summary of change

The feature was created so it works as it is shown on screenshots
For new design:
image
image

For old design:
image
image

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: app/views/calculators/calculator.html.slim [hunk 1]
```diff
@@ -1,84 +1,176 @@
...
-        .flex-item.flex-row.rounded.w-100.age_wrapper.form_fild
-          = form.input_field :years,
-                              collection: 0..2,
-                              prompt:'__',
-                              class:'select-age ml-2 rounded',
-                              data: { input_child_info_target: 'year', action:'input-child-info#yearChanged' }
+- if Flipper[:old_calculator_design].enabled?
+  .jumbotron
+    .container-fluid
+      a.rounded.float-right.calculator__cross__space href=root_path
```
Comment (mykhailovasylenchuk) [inline app/views/calculators/calculator.html.slim:4]: link_to

File: app/views/calculators/calculator.html.slim [hunk 2]
```diff
@@ -1,84 +1,176 @@
...
-        = t '.money_spent'
 
-.container.calculation-results
-  .row
-    .col-xl-8.result-card
-      = render 'layouts/description_block'
-    .col-sm.result-card
-      button.calculate-btn.description-btn
-        = render 'layouts/description_button'
+          / = form.text_field  :baby_weight, placeholder:t('.form.baby_weight_label')
```
Comment (mykhailovasylenchuk) [inline app/views/calculators/calculator.html.slim:128]: Коменти видаляй якщо не треба

File: app/views/calculators/calculator.html.slim [hunk 3]
```diff
@@ -1,84 +1,176 @@
...
+                  option
+                    =t('calculators.calculator.form.budgetary')
+                  option selected=""
+                    =t('calculators.calculator.form.medium')
+                  option
+                    =t('calculators.calculator.form.premium')
+            = form.text_field :baby_weight, placeholder:t('.form.baby_weight_label'), class:"form-input d-none"
+            = form.text_field :diapers_per_day, placeholder: "Diapers per day", class:"form-input d-none"
+          / div.flex-item.mt-4
+          /   = simple_format("", { type: "checkbox", name: "email_receiver", id: "checkbox_submit" },
```
Comment (mykhailovasylenchuk) [inline app/views/calculators/calculator.html.slim:43]: Тут теж

Comment (BarryLev) [reply app/views/calculators/calculator.html.slim:43]: done

File: app/views/calculators/calculator.html.slim (hunk 2 above)
Comment (BarryLev) [reply app/views/calculators/calculator.html.slim:128]: done

File: app/views/calculators/calculator.html.slim [hunk 4]
```diff
@@ -1,84 +1,152 @@
...
+              p.form-a
+                = t '.form_price'
+                = select_tag 'product_category',
+                  options_for_select([[t('calculators.calculator.form.budgetary'), nil],
+                    [t('calculators.calculator.form.medium'), 'medium'],
+                    [t('calculators.calculator.form.premium'), 'premium']],
+                  { selected: 'medium' }), { class: 'custom-select', required: true }
+            = form.text_field :baby_weight, placeholder:t('.form.baby_weight_label'), class:"form-input d-none"
+            = form.text_field :diapers_per_day, placeholder: "Diapers per day", class:"form-input d-none"
+          button.calculate-btn.result-btn[type="submit" id="button_submit"]
```
Comment (mykhailovasylenchuk) [inline app/views/calculators/calculator.html.slim:31]: button_tag

File: app/views/calculators/calculator.html.slim [hunk 5]
```diff
@@ -1,84 +1,152 @@
...
+        span
+          p [data-results-target="moneySpent"] 0
+          = t '.money_spent'
+
+  .container.calculation-results
+    .row
+      .col-xl-8.result-card
+        = render 'layouts/description_block'
+      .col-sm.result-card
+        button.calculate-btn.description-btn
```
Comment (mykhailovasylenchuk) [inline app/views/calculators/calculator.html.slim:150]: button_tag

**Review by mykhailovasylenchuk — APPROVED** [review]

File: app/views/calculators/calculator.html.slim [hunk 6]
```diff
@@ -1,84 +1,150 @@
...
-      .flex-item.flex-column
-        .input_lable
-          = t('.form_description')
-        .flex-item.flex-row.rounded.w-100.age_wrapper.form_fild
-          = form.input_field :years,
-                              collection: 0..2,
-                              prompt:'__',
-                              class:'select-age ml-2 rounded',
-                              data: { input_child_info_target: 'year', action:'input-child-info#yearChanged' }
+- if Flipper[:old_calculator_design].enabled?
```
Comment (loqimean) [inline app/views/calculators/calculator.html.slim:1]: рознеси по паршалам, бо це вже ту матч файл, най буде щось типу старий-новий дизайн паршали, щоб то було простіше менеджити

File: app/views/calculators/calculator.html.slim (hunk 4 above)
Comment (BarryLev) [reply app/views/calculators/calculator.html.slim]: done

File: app/views/calculators/calculator.html.slim (hunk 5 above)
Comment (BarryLev) [reply app/views/calculators/calculator.html.slim]: done

**Pull request closed without merging**